src/components/TipInput.vue

```diff
@@ -217,6 +228,11 @@ export default {
       min-width: 16rem;
       padding: 0.5rem;
     }
+
+    .not-bootstrap-modal-content {
```

Comment:
I can't find `.not-bootstrap-modal-content` inside `.not-bootstrap-modal-content` inside `.not-bootstrap-modal`. Is this styles used somewhere?

Reply:
Yes, the dropdown contains a < Modal > component and it is within one so it is a one time case where we have dropdown within a dropdown. This style overrides the parent one.

Reply:
Ok, I got that. But, actually, you can remove this block by making the selector more specific as I did in 2311feb04d35a81c7aa07b201903f5dd223420c6